modules/library/src/main/scala/zio/elasticsearch/executor/response/SearchWithAggregationsResponse.scala

```diff
@@ -55,9 +55,8 @@ private[elasticsearch] final case class SearchWithAggregationsResponse(
       )
       .toEitherWith(_.mkString(", "))
 
-  lazy val results: List[Json] = hits.hits.map(_.source)
-
-  lazy val resultsWithHighlights: List[(Json, Option[Json])] = hits.hits.map(h => (h.source, h.highlight))
+  lazy val resultsWithHighlightsAndSort: List[(Json, Option[Json], Option[Json])] =
```

Comment:
We should think about a dedicated type for the tuple `(Json, Option[Json], Option[Json])` so we wouldn't need to communicate the meaning of each element through a complicated value name.

Reply:
Yes definetly. It would be much easier to understand if we had case class for this. I will implement it, just not sure about naming of such class.